### PR TITLE
express-post-json | Wonseop Lee

### PR DESCRIPTION
**Pull request by wonseop-lee**
express-post-json

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const arr = [];
```
Comment (BrettAlbright) [inline express-post-json/index.js]: This array should only exist in the get request. It should be created new each time and sent to the client with the current grades objects

File: express-post-json/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+});
+
+app.use(express.json());
+
+app.post('/api/grades', function(req, res) {
+  res.status(201);
+  let grades = req.body;
+  grades.id = nextId;
+  nextId++;
+  arr.push(grades);
```
Comment (BrettAlbright) [inline express-post-json/index.js]: remove this

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small updates needed!

**Comment by wonseop-lee** [comment]
express-post-json

**Comment by wonseop-lee** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**